src/index.css

```diff
@@ -19,5 +19,8 @@ code {
 main {
   display: flex;
   flex-direction: row;
+  justify-content: center;
   width: 100%;
+  min-height: calc(100vh - 227px);
```

Comment:
Used to calculate the height to be everything except for the AppBar and AppFooter when the screen size is too large for the content inside the main tag